types/vote_test.go

```diff
@@ -47,9 +48,11 @@ func exampleVote(t byte) *Vote {
 
 func TestVoteSignable(t *testing.T) {
 	vote := examplePrecommit()
-	signBytes := vote.SignBytes("test_chain_id")
+	v := vote.ToProto()
+	signBytes := VoteSignBytes("test_chain_id", v)
+	pb := CanonicalizeVote("test_chain_id", v)
 
-	expected, err := cdc.MarshalBinaryLengthPrefixed(CanonicalizeVote("test_chain_id", vote))
```

Comment:
This is also relevant to the ledger lib that @jleni wrote as it will expect this length prefix 😬 
I'm actually not sure if someone uses the ledger to sign votes etc but I guess as it is possible, this should come with a note in the changelog.